src/core/param/param.h

```diff
@@ -160,6 +160,15 @@ struct Param {
   ///     leaking_edges = true
   bool leaking_edges_ = true;
 
+  /* A string for determining diffusion type within the simulation space.
+  current inputs include Euler and RK
+
+          [simulation]
+          diffusion_type_ = "Euler"
+  */
+
+  std::string diffusion_type_ = "Euler";
```

Comment:
Use the same comment style as all the other parameters.
The code in `param.cc` were the toml file is parsed is missing.
Please also add a test in [1]  where you set the parameter to a non-default value to check if parsing works.

[1] https://github.com/BioDynaMo/biodynamo/blob/master/test/unit/core/simulation_test.cc#L31-L82

Reply:
Please address these two comments: commenting style + parameter test

Reply:
These two comments have been addressed within the param.h, param.cc and test for [1] within the Runge-Kutta branch, I am unsure why this comment is not marked as outdated ? 

https://github.com/BioDynaMo/biodynamo/blob/f459c0b9f330be02e1cb4649b873393654b687f9/src/core/param/param.h#L163

https://github.com/BioDynaMo/biodynamo/blob/f459c0b9f330be02e1cb4649b873393654b687f9/src/core/param/param.cc#L90

https://github.com/BioDynaMo/biodynamo/blob/f459c0b9f330be02e1cb4649b873393654b687f9/test/unit/core/simulation_test.cc#L43